### ISSUE TITLE
Nikola serve lists 0.0.0.0:8000 but actually served at http://127.0.0.1:8000/

### ISSUE THREAD

**Issue by jhconning** [getnikola/nikola#2755]
### Environment 
**Python Version:** 3.6.1
**Nikola Version:** Nikola (7.8.4)
**Operating System:** Windows 10
### Description:  When I type `nikola serve --browser` a page is opened at 0.0.0.0:8000 but it returns a 'can't be reached' error (ERR_ADDRESS_INVALID).  The site is however being correctly served at http://127.0.0.1:8000/

I'm new to Nikola I've really just followed the demo (choosing defaults for most values).  

```
G:\E\nikola\demo>nikola serve --browser
[2017-05-11T18:01:40Z] INFO: serve: Serving HTTP on 0.0.0.0 port 8000...
[2017-05-11T18:01:40Z] INFO: serve: Opening http://0.0.0.0:8000/ in the default web browser...
```
This opens up this page:

![image](https://cloud.githubusercontent.com/assets/5048607/25964565/499e7b0c-3653-11e7-8918-72ab8cda9acb.png)

The site is however being served correctly to  http://127.0.0.1:8000/

![image](https://cloud.githubusercontent.com/assets/5048607/25964961/93a562b4-3654-11e7-91a6-82230e6da174.png)

Did something recently change or is something in my configuration redirecting? Thanks

p.s. -- http://127.0.0.1:8000/ and http://127.1.1.1:8000/ both serve the site.
p.p.s -- edited issue title and body to fix address (I meant http://127.0.0.1:8000/ not http://127.1.1.1:8000/

**Comment by Kwpolska** [comment]
Does `127.0.0.1` work? Does `0.0.0.0:8000` work after a refresh? Does it work in a different browser?

(We *do* use `0.0.0.0`, and on Linux/macOS it works — it is the “all local interfaces” IP)

**Comment by jhconning** [comment]
Yes, `127.0.0.1` also works.   I tried a refresh on `0.0.0.0` but that did not work. I'm on Windows 10.

**Comment by Kwpolska** [comment]
Does any other browser work?

**Comment by jhconning** [comment]
I've tried `0.0.0.0' on chrome (my default) and firefox ('unable to connect')

**Comment by Kwpolska** [comment]
Explanation: https://superuser.com/questions/536156/how-do-i-get-0-0-0-0-to-resolve-to-localhost-when-browsing-a-url-that-contains-0

Fix: #2756 — testing welcome.

**Comment by jhconning** [comment]
Thank you for the fultra fast response and fix!   It works now, opening a browser window on `127.0.0.1:8000`

```
nikola serve --browser
[2017-05-12T23:55:39Z] INFO: serve: Serving HTTP on 0.0.0.0 port 8000...
[2017-05-12T23:55:39Z] INFO: serve: Opening http://127.0.0.1:8000/ in the default web browser...
```
